keps/sig-windows/2578-windows-conformance/README.md

```diff
@@ -226,8 +226,8 @@ Nevertheless, we define this specification because it is an obvious starting poi
 #### Active Directory
 
 - Ability to run a pod as a GMSA User with only be able to access allowed resources
-- Ability to read and write from local storage using ActiveDirectory credentials
-- Observed lack of Ability to read and write from local storage when ActiveDirectory protected resources credentials are not present
+- Ability to read and write from remote folder using ActiveDirectory credentials
```

Comment:
Can we test reading from/writing to local and remote storage?

Reply:
The local storage test is covered by this test case ([here](https://github.com/kubernetes/enhancements/commit/3ee6784895d650caa99e3ad23a361465478effb5#diff-bbb06393cef572c658f3bd68871c382e7e755c238e786d17456e18d371e0ba45R185-R186)) and it's not related to active directory. The test case here wants to test the ability to read/write files to a folder located on active directory (a separate vm) from a windows node.

Reply:
Can we include a test case that validates we can read to/write file a local file share using ActiveDirectory creds and also have a test case that validates we can read/write to a remote file share?
Both scenarios are useful and not covered by the local storage test case you linked.

Reply:
Ah I see, I'll follow up with you after trying the local storage test case.

Reply:
Hi Mark! Nice talking to you today. I updated the code and pr description based on our meeting.